tox.ini

```diff
@@ -2,13 +2,17 @@
 envlist =
     csslint
     eslint
-    py36
+    py{36}-django{111,20,21,22}
```

Comment:
Should probably test with py35 instead, as that's what we currently use in production.

Reply:
Done.